### ISSUE TITLE
[Feature Request] Do not invert coloured emoji

### ISSUE THREAD

**Issue by thany** [darkreader/darkreader#10134]
# Feature Request

## Is this feature request related to a problem?
Inverted emoji look weird.

The regular emoji would look much better in a dark mode setting. On top of that, you don't see many websites (meaning probably close to none) that also invert emoji in their own dark modes. So, I would suggest leaving emoji as they are, as well as any other glyphs that are given a colour from within the font itself. Only invert regular text and other non-coloured glyphs.

It might be best to add this as a per-site option.

## Screenshots
I hope this simple example makes it clear what I mean.

Please compare this:
![image](https://user-images.githubusercontent.com/152227/196726230-f74ea01c-346b-42b4-89d1-f26d14a28d94.png)
To this:
![image](https://user-images.githubusercontent.com/152227/196726328-b50fdd2a-5af8-4aed-bfbe-52a9f87a5c9c.png)

## Additional Context
Running Firefox 106 on Windows 10.

**Comment by bershanskiy** [comment]
@thany Thanks for your report! Could you please link to a site where this happens? One possible reason is that those images are not actually emojis (glyps, symbols), but are actually just images (icons, vector or raster images). If you could provide a link, I would be able to tell Dark Reader to not invert these images.

For reference, here is a bunch of emojis: https://getemoji.com/
Most of them are not inverted since they are rendered by the browser and do not support CSS modifiers like `color`. There are a few exceptions like ☺️ which for me look like this: 
![image](https://user-images.githubusercontent.com/45960703/196908948-2ca268fd-68c7-4cf4-9e44-fbf3e6411469.png)

The modified symbols are just simple monochrome glyps which do support CSS `color`.

**Comment by thany** [comment]
It's Jira, and the one we use is an internal-only instance. I think any other Jira instance will do as well.

I just checked the DOM, and it is definitely not an image:
![image](https://user-images.githubusercontent.com/152227/197179217-645ce30e-2757-4960-aaa0-e2f6444568b5.png)

**Comment by bershanskiy** [comment]
Thanks for more info, I'll look into it.

This is curious: Chromium now displays this emoji properly for me. I don't remember updating system fonts and my Chromium package version was released on October 14. I wonder what changed.
![image](https://user-images.githubusercontent.com/45960703/197286611-6d193e9a-c12c-47de-9160-24324fe76ac5.png)

**Comment by thany** [comment]
Yes, here on Github everything's good. But on a Jira instance I get those inverted coloured emoji.

I've seen a few other sites where coloured emoji get inverted, but I don't remember which ones. Just saying it's not exclusive to Jira.

**Comment by mmuelle4** [comment]
Happens in Google docs as well.

<img width="212" alt="image" src="https://github.com/darkreader/darkreader/assets/13357267/eb184222-84c5-4c60-8e4a-e221fde2234f">

But... I just checked the DOM and it looks like the entire gdoc is a `<canvas>` element, so that's probably not feasible to fix.

**Comment by alastair-games** [comment]
I'm getting this bug with Google Sheets too, the emoji are inverted. 

Is there way to make an exception for them somehow?
